go.mod

```diff
@@ -7,85 +7,75 @@ replace (
+	k8s.io/apiserver => k8s.io/apiserver v0.26.10
+	k8s.io/cli-runtime => k8s.io/cli-runtime v0.26.10
+	k8s.io/client-go => k8s.io/client-go v0.26.10
+	k8s.io/cloud-provider => k8s.io/cloud-provider v0.26.10
+	k8s.io/cluster-bootstrap => k8s.io/cluster-bootstrap v0.26.10
+	k8s.io/code-generator => k8s.io/code-generator v0.26.10
+	k8s.io/component-base => k8s.io/component-base v0.26.10
+	k8s.io/component-helpers => k8s.io/component-helpers v0.26.10
+	k8s.io/controller-manager => k8s.io/controller-manager v0.26.10
+	k8s.io/cri-api => k8s.io/cri-api v0.26.10
+	k8s.io/csi-translation-lib => k8s.io/csi-translation-lib v0.26.10
+	k8s.io/dynamic-resource-allocation => k8s.io/dynamic-resource-allocation v0.26.10
+	k8s.io/kms => k8s.io/kms v0.26.10
+	k8s.io/kube-aggregator => k8s.io/kube-aggregator v0.26.10
+	k8s.io/kube-controller-manager => k8s.io/kube-controller-manager v0.26.10
+	k8s.io/kube-proxy => k8s.io/kube-proxy v0.26.10
+	k8s.io/kube-scheduler => k8s.io/kube-scheduler v0.26.10
+	k8s.io/kubectl => k8s.io/kubectl v0.26.10
+	k8s.io/kubelet => k8s.io/kubelet v0.26.10
+	k8s.io/kubernetes => k8s.io/kubernetes v1.26.10
+	k8s.io/legacy-cloud-providers => k8s.io/legacy-cloud-providers v0.26.10
+	k8s.io/metrics => k8s.io/metrics v0.26.10
+	k8s.io/mount-utils => k8s.io/mount-utils v0.26.10
+	k8s.io/pod-security-admission => k8s.io/pod-security-admission v0.26.10
+	k8s.io/sample-apiserver => k8s.io/sample-apiserver v0.26.10
 
-	kubevirt.io/api => github.com/kubevirt/api v0.54.0
-	kubevirt.io/client-go => github.com/kubevirt/client-go v0.54.0
-	launchpad.net/gocheck v0.0.0-20140225173054-000000000087 => github.com/go-check/check v0.0.0-20180628173108-788fd7840127
-	sigs.k8s.io/cluster-api => sigs.k8s.io/cluster-api v1.1.4
+	kubevirt.io/api => kubevirt.io/api v1.0.0
+	kubevirt.io/client-go => kubevirt.io/client-go v1.0.0
 	sigs.k8s.io/structured-merge-diff => sigs.k8s.io/structured-merge-diff v0.0.0-20190302045857-e85c7b244fd2
 )
 
 require (
 	github.com/Masterminds/semver/v3 v3.2.0
-	github.com/banzaicloud/logging-operator/pkg/sdk v0.8.16
-	github.com/banzaicloud/operator-tools v0.28.10
+	github.com/cisco-open/operator-tools v0.29.0
 	github.com/containernetworking/cni v1.1.2
-	github.com/docker/go-units v0.4.0
+	github.com/docker/go-units v0.5.0
 	github.com/ehazlett/simplelog v0.0.0-20200226020431-d374894e92a4
 	github.com/emicklei/go-restful/v3 v3.9.0
-	github.com/gobuffalo/flect v0.2.5
+	github.com/gobuffalo/flect v1.0.2
 	github.com/gorilla/mux v1.8.0
 	github.com/guonaihong/gout v0.1.3
 	github.com/harvester/harvester-network-controller v0.3.1
 	github.com/harvester/node-manager v0.1.5-0.20230614075852-de2da3ef3aca
 	github.com/iancoleman/strcase v0.2.0
 	github.com/k3s-io/helm-controller v0.11.7
-	github.com/k8snetworkplumbingwg/network-attachment-definition-client v0.0.0-20200331171230-d50e42f2b669
+	github.com/k8snetworkplumbingwg/network-attachment-definition-client v1.3.0
+	github.com/kube-logging/logging-operator/pkg/sdk v0.9.1
```

Comment:
@starbops It looks like `banzaicloud/logging-operator` has been changed to `kube-logging/logging-oeprator`. Could you check whether it's safe to do this change in `go.mod`? Thank you.

There is a function `loggingWorkaround`. Not sure whether we need do any update for it?

https://github.com/harvester/harvester/blob/acb6e69121a9843c8e7616de9f44401293c4d238/pkg/codegen/main.go#L200-L214

Reply:
The GroupVersion has not changed, so keeping it untouched is okay. Thank you.